react/containers/NavBar/view.jsx

```diff
@@ -63,17 +71,18 @@ class NavBar extends React.Component {
             <span className="nav-bar-tagline">Open-source, decentralized image and video sharing.</span>
           </div>
           <div className="nav-bar--right">
-            <a className="nav-bar-link link--nav-active" href="/">Publish</a>
-            <a className="nav-bar-link link--nav" href="/about">About</a>
+            <NavLink className="nav-bar-link link--nav" activeClassName="link--nav-active" to="/" exact={true}>Publish</NavLink>
```

Comment:
I'm not quite sure I understand.  Do you mean create my own `NavLink` component ( rather than using the `NavLink` from `react-router-dom`) and control the className from inside the component?

Reply:
Oh I didn't realize that was a `react-router-dom` component. But yeah, I would make a custom `NavLink` component that uses the `react-router-dom` component. Then you can extend the functionality/style very easily.

Reply:
similar to this https://github.com/lbryio/lbry-app/blob/redesign-wip/src/renderer/component/link/view.jsx